index.html

```diff
@@ -107,6 +108,182 @@
             <strong>Error Alert</strong>
         </div>
     </section>
+
+    <!-- CARDS -->
+    <div class="container">
+        <figure class="content main">
+            <img src="https://static.nike.com/a/images/t_PDP_864_v1/f_auto,b_rgb:f5f5f5/3cd6ba48-cf35-4439-b0d5-0fe40ca05eb6/air-max-2021-shoes-MljWq1.png"
+                alt="Image of bubbles through a microscope." />
+            <figcaption class="title">
+                <h2>Nike Air Max</h2>
+                <p>Soft and Smooth.</p>
+            </figcaption>
+        </figure>
+        <div class="about">
+            <h5 class="product-description">Sustainable Materials</h5>
+            <p></p>
+            <p>Unisex</p>
+            <p>available in colors</p>
+            <div id="colors">
+                <div id="blue"></div>
+                <div id="green"></div>
+                <div id="red"></div>
+            </div>
+            <h5 class="product-price">
+                ₹ 8,995
+            </h5>
+        </div>
+        <div class="icons">
+            <button class="btn_otl_default">Add To Cart</button>
+            <i class="far fa-heart"></i>
+            <i class="fas fa-share-alt"></i>
+            <i class="fas fa-ellipsis-v"></i>
+        </div>
+    </div>
+
+    <div class="container">
+        <figure class="content badge">
+            <img src="https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=880&q=80"
+                alt="Image of bubbles through a microscope." />
+            <figcaption class="title">
+                <h2>Nike Air Max</h2>
+                <p>Soft and Smooth.</p>
+            </figcaption>
+            <i class="far fa-heart"></i>
+            <h4>NEW</h4>
+        </figure>
+        <div class="about">
+            <h5 class="product-description">Sustainable Materials</h5>
+            <p></p>
+            <p>Unisex</p>
+            <p>available in colors</p>
+            <div id="colors">
+                <div id="blue"></div>
+                <div id="green"></div>
+                <div id="red"></div>

```

Comment:
`id` attribute is supposed to be unique. If you want to use it multiple times you should use `class`

Reply:
hey Kartikeya thanks for your reviews will make the changes right away 🚀